src/Jackett.Common/Definitions/torrentmasters.yml

```diff
@@ -26,8 +26,8 @@ caps:
     - {id: 95, cat: PC, desc: "Programok - Egyéb (Win)"}
     - {id: 74, cat: PC/ISO, desc: "Programok - ISO (Win)"}
     - {id: 75, cat: PC/Mac, desc: "Programok - Linux/Unix/Mac OS"}
-    - {id: 10, cat: TV/SD, desc: "Sorozat - DVD (ENG)"}
-    - {id: 10, cat: TV/SD, desc: "Sorozat - DVD (HUN)"}
+    - {id: 85, cat: TV/SD, desc: "Sorozat - DVD (ENG)"}
+    - {id: 66, cat: TV/SD, desc: "Sorozat - DVD (HUN)"}
```

Comment:
Hmm. 85 and 66 are already in use as the ids for the movies/dvd 
I suggest you review ALL your categories one more time ;-)

Reply:
😣 Thanks for being so patient, no idea why I wrote 10 there originally.